recipes/gwframes/meta.yaml

```
@@ -9,13 +9,13 @@ source:
   # we have to use git_url here to get the submodules needed for the
   # build; see <https://github.com/conda-forge/xgboost-feedstock/issues/22>
   git_url: https://github.com/moble/{{ name }}.git
-  git_rev: 792a341f77fa838bcdfd78940850205bd60c5ba4
+  git_rev: {{ version }}
   # Mitigate the above by just cloning the minimum depth
   git_depth: 1
 
 build:
   number: 0
-  script: "{{ PYTHON }} -m pip install ./Code/"
+  script: "cd Code && {{ PYTHON }} setup.py install"
```

Sorry about that. Can you try:
```suggestion
  script: "cd Code && {{ PYTHON }} -m pip install ."
```